src/components/editor/Editor.svelte

```diff
@@ -83,6 +84,9 @@
     import Menu, { RevisionSet } from './util/Menu';
     import { OutlinePadding } from './util/outline';
 
+    // Local deletion notification state (instead of a global store)
```

Comment:
State should be declared below the props interface, and preferably nearby other related state, as the component has become quite large. Organization really matters here, to help others make sense of it.